service/src/main/java/uk/gov/hmcts/reform/fpl/service/SendDocumentService.java

```diff
@@ -37,12 +37,20 @@
     private final SentDocumentHistoryService sentDocuments;
 
     public void sendDocuments(CaseData caseData, List<DocumentReference> documentToBeSent, List<Recipient> parties) {
-        sendDocuments(caseData,documentToBeSent,parties,Language.ENGLISH);
+        sendDocuments(new SendDocumentRequest(caseData,
+            defaultIfNull(documentToBeSent, new ArrayList<DocumentReference>()).stream()
+                .map(doc -> DocumentReferenceWithLanguage.builder()
```

Comment:
could have a static method called `from` in `DocumentReferenceWithLanguage` that just accepts a `DocumentReference` and then builds things there